src/gateway/codeship.js

```diff
@@ -0,0 +1,61 @@
+require('dotenv').load();
```

Comment:
this should be moved to the config.js file, and should only happen when the NODE_ENV is development